docs/20-guides/101-datocms/index.md

```diff
@@ -0,0 +1,272 @@
+---
+title: DatoCMS Integration Guide
+description: Learn how to create a data model in DatoCMS and integrate it with Nordcraft.
+---
+
+# DatoCMS Integration Guide
+
+DatoCMS is a headless Content Management System (CMS) that lets you store and manage content, and retrieve that content via a GraphQL API for use in your front-end applications. This guide outlines the process of creating a data model within DatoCMS and integrating it with Nordcraft.
+
+### Source Projects
+
+Clone the projects below to your DatoCMS and Nordcraft accounts to follow along with this guide:
+
+*   [DatoCMS Sample Project](https://dashboard.datocms.com/clone?projectId=161783&name=Nordcraft%20Sample)
```

Comment:
To follow conventions in the rest of the repo, lists should be marked with `-` and not -*-